go.mod

```diff
@@ -5,6 +5,9 @@ go 1.12
 require (
 	github.com/client9/misspell v0.3.4
 	github.com/google/addlicense v0.0.0-20190907113143-be125746c2c4
+	github.com/open-telemetry/opentelemetry-collector v0.2.1-0.20191016224815-dfabfb0c1d1e
```

Comment:
Can we have this module together with main.go? This will follow the same pattern that we have for individual components.

Reply:
Good point. I was planning on doing it if/when we introduce multiple builds from this repo (protobuf vs gogoproto) but will look into doing it right away. We'll have to keep this go.mod around anyway so this repo can be consumed as a lib easily (stuff other than explicit modules like receivers, exporters).

Reply:
On second thought, I think we should punt it for now. If we make it a module, it'll not be able to access any `internal` package from the rest of the repo. The cmd module will by default import code from Github master instead of the local code so local development will be annoying. We can add permanent replace directives but I think there will be more issues. We'll mostly need to update the Makefile, CI and some scritps to supports it. Added a ticket to track it: #35

Reply:
Sounds good.